### ISSUE TITLE
PMID:29900664

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2551]
We found that the longevity induced by tschimganine was dependent on the Sty1 pathway

but we have
	stationary phase viability unaffected by alpha-hibitakanine
sty1Δ

**Comment by ValWood** [comment]
WT cells live longer when exposed to  tschimganine

mutant cells don't live longer

therefore 
should it be
stationary phase viability decreased by alpha-hibitakanine?

**Comment by mah11** [comment]
> should it be
> stationary phase viability decreased by alpha-hibitakanine?

No, that would suggest that mutants live for a shorter time in the presence of that stuff than without it, which isn't what happens. The wild-type behavior is to live longer when exposed to alpha-hibitakanine; the mutant does not "respond" by living longer as wild type does.

iow, these all have the same lifespan:
- wt - stuff
- mutant - stuff
- mutant + stuff

... whereas wt + stuff lives longer. I gave this a good think when I did the approval for this paper.
